install.md

```diff
@@ -174,47 +264,32 @@ As with other Go projects, PODMAN must be cloned into a directory structure like
 GOPATH
 └── src
     └── github.com
-	└── containers
-	    └── libpod
-```
-
-First, configure a `GOPATH` (if you are using go1.8 or later, this defaults to `~/go`)
-and then add $GOPATH/bin to your $PATH environment variable.
-
-```bash
-export GOPATH=~/go
-mkdir -p $GOPATH
-export PATH=$PATH:$GOPATH/bin
+        └── containers
+            └── libpod
 ```
 
-Next, clone the source code using:
+First, ensure that the go version that is found first on the $PATH (in case you built your own; see above) is sufficiently recent -
+`go version` must be higher than 1.10.x). Then we can finally build podman (assuming we already have a `$GOPATH` and the corresponding folder,
+`export GOPATH=~/go && mkdir -p $GOPATH`):
 
 ```bash
-mkdir -p $GOPATH/src/github.com/containers
-cd $_ # or cd $GOPATH/src/github.com/containers
-git clone https://github.com/containers/libpod # or your fork
-cd libpod
```

Comment:
Replaced with the build instructions from the tutorial, and added `BUILDTAGS="selinux seccomp"` due to the discussion in #3045. I also needed to add `PREFIX=` to make sure all the libraries get linked into the binary correctly (although I'd hope that this could be removed again somehow).